build_tools/configure_stm32f4.sh

```diff
@@ -75,13 +75,31 @@ if ! [ -d "$PATH_TO_IREE_HOST_BINARY_ROOT/bin/" ]; then
 fi
 
 # Configure project
-cmake -GNinja \
-      -DBUILD_WITH_CMSIS=ON \
-      -DCMAKE_TOOLCHAIN_FILE="`realpath ../build_tools/cmake/arm.toolchain.cmake`" \
-      -DARM_TOOLCHAIN_ROOT="${PATH_TO_ARM_TOOLCHAIN}" \
-      -DARM_CPU="armv7e-m" \
-      -DIREE_HAL_DRIVERS_TO_BUILD="VMVX_Sync;DYLIB_Sync" \
-      -DIREE_HOST_BINARY_ROOT="${PATH_TO_IREE_HOST_BINARY_ROOT}" \
-      -DCUSTOM_ARM_LINKER_FLAGS="${CUSTOM_ARM_LINKER_FLAGS}" \
-      -DLINKER_SCRIPT="${PATH_TO_LINKER_SCRIPT}" \
-      ..
+cmake_args=()
+cmake_args+=(-GNinja)
+
+case $1 in
+  cmsis)
+    cmake_args+=(-DBUILD_WITH_CMSIS=ON)
+    ;;
+
+  libopencm3)
+    cmake_args+=(-DBUILD_WITH_LIBOPENCM3=ON)
+    ;;
+
+  *)
+    echo "Unknown lib. Use 'cmsis' or 'libopencm3"
+    exit 1
+    ;;
+esac
```

Comment:
No need to add another/duplicate the case statement. We already have one that checks for cmsis/libopencm3. We can extend the existing one to add a new bash variable or we add an element to the array there (However, I see no strong benefit of switching to an array).

Reply:
In any case, thanks for checking more carefully than we did and for taking over to fix.